src/main/resources/db/migration/V7__create_auth_tables.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE `Role`(
+    RoleID TINYINT NOT NULL,
+    Name varchar(64) NOT NULL,
+    PRIMARY KEY (RoleID)
+);
+
+INSERT INTO `Role` VALUES (1, 'HR');
+INSERT INTO `Role` VALUES (2, 'Sales');
+INSERT INTO `Role` VALUES (3, 'Management');
+
+CREATE TABLE `User` (
+    Username varchar(64) NOT NULL,
```

Comment:
should this be a unique constraint?

Reply:
It could be added, but username is the primary key. We could also create a userId and use it as the PK?